packages/keyring-eth-trezor/src/trezor-connect-bridge.ts

```diff
@@ -35,29 +40,36 @@ export class TrezorConnectBridge implements TrezorBridge {
     this.trezorConnectInitiated = true;
   }
 
-  dispose() {
+  async dispose(): Promise<void> {
```

Comment:
Was not `async` but since it's returning a `Promise` it makes sense to make it `async`